cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help

```diff
@@ -22,8 +22,8 @@ OPTIONS
 
 
 EXAMPLES
-    Add a module with a main
+    Add a module with a main:
     $ ballerina add mainmodule
```

Comment:
```suggestion
    $ ballerina add mymain
```
Changed the module name in case the user gets the wrong impression to get a module with a main you have to run the given command exactly